cli/template/base/tsconfig.json

```diff
@@ -31,7 +31,7 @@
     }
   },
   "include": [
-    ".eslintrc.cjs",
+    "eslint.config.js",
```

Comment:
```suggestion
```

Don't need this at all now, it's captured by the glob below

(Files starting with . aren't matched by glob which is why the explicitness was needed before)

Reply:
OK I see